types/qunit/index.d.ts

```diff
@@ -559,6 +559,12 @@ declare global {
             assert: Assert;
         }
 
+        interface TestFunction {
```

Comment:
I approved but if we can add `only` and `todo` for completeness that would be 🔥

Reply:
@gitKrystan I added types for:
```typescript
test.skip
test.todo
test.only
test.if

test.each
test.todo.each
test.only.each
test.if.each
test.skip.each
```